app/models/activities/fetcher.rb

```diff
@@ -57,6 +57,7 @@ def event_types
         else
           OpenProject::Activity.available_event_types.to_a
         end
+      .excluding('budgets')
```

Comment:
The cleaner way is to remove `Activities::BudgetActivityProvider` and the registration of it in the budget's engine.

I thought about also removing the journals while we are at it on account of them not carrying a lot of valuable information but that might be too hasty.